python/ray/serve/_private/http_proxy.py

```diff
@@ -80,6 +81,9 @@
     os.environ.get("SERVE_SOCKET_REUSE_PORT_ENABLED", "1") == "1"
 )
 
+RAY_SERVE_HTTP_KEEPALIVE_TIMEOUT_S = int(
+    os.environ.get("RAY_SERVE_HTTP_KEEPALIVE_TIMEOUT_S", 0)
```

Comment:
```suggestion
RAY_SERVE_HTTP_KEEP_ALIVE_TIMEOUT_S = int(
    os.environ.get("RAY_SERVE_HTTP_KEEP_ALIVE_TIMEOUT_S", 0)
```

Reply:
I think these are used interchangeably, but in this PR you're using `keep_alive` everywhere else

Reply:
oops, thanks!